### PR TITLE
Update Algorithm Slasher Flick

### PR DESCRIPTION
**Pull request by dakshshah96**
Refer #1094

### CODE REVIEW

File: Algorithm-Slasher-Flick.md [hunk 1]
```diff
@@ -1,77 +1,94 @@
 # Algorithm Slasher Flick
 
-### Problem Explanation:
+:triangular_flag_on_post: Remember to use [**`Read-Search-Ask`**](FreeCodeCamp-Get-Help) if you get stuck. Try to pair program :busts_in_silhouette: and write your own code :pencil:
 
-Return the remaining elements of an array after chopping off n elements from the head.
+### :checkered_flag: Problem Explanation:
 
-For example: `slasher([1, 2, 3], 2);` must return [3]
+For example: `slasher([1, 2, 3], 2);` must return [3].
```
Comment (atjonathan) [inline Algorithm-Slasher-Flick.md]: Add general "Relevant Links"

**Comment by Rafase282** [comment]
Please add these links to the general part.

Array.slice()
Array.splice()

**Comment by dakshshah96** [comment]
Suggestions implemented.

**Comment by Rafase282** [comment]
# LGTM :+1:

**Pull request merged**